ZupIT/ritchie-cli
# Added tests to update repo command

**- What I did**
Hi Ritchiers,
Add tests for the rit update repo command

> Master
> ![master](https://user-images.githubusercontent.com/14079465/92938376-d6523a80-f422-11ea-961f-74392b5a6d41.png)
> ->
> PR
>![pr](https://user-images.githubusercontent.com/14079465/92938677-3fd24900-f423-11ea-9ddb-e3a411762163.png)



**- How to verify it**
> Clone this pr with the command
```bash
git fetch upstream pull/515/head:improvement/added-tests-to-update-repo-command

```
> And run goimports
```bash
git checkout master
make build && cp dist/linux/rit /usr/local/bin/
go test -v -failfast -short -coverprofile=bin/cov.out ./pkg/cmd && go tool cover -func=bin/cov.out # Run unit tests of the cmd pkg

git checkout improvement/added-tests-to-update-repo-command
make build && cp dist/linux/rit /usr/local/bin/
go test -v -failfast -short -coverprofile=bin/cov.out ./pkg/cmd && go tool cover -func=bin/cov.out #  Run and see the differences
```

**- Description for the changelog**

Added tests to update repo command
